flybywiresim/a32nx
# chore: upgrade msfssdk to 0.4.0, remove fork

Successor to #7586 

## Summary of Changes

This patch ports the PFD, EWD and ATSU to use `msfssdk` from npm, and removes our fork of the framework.

## Testing instructions


- Ensure all PFD indications function correctly
- Ensure all EWD indications function correctly
- Ensure all ATSU indications function correctly


## How to download the PR for QA

Every new commit to this PR will cause a new A32NX artifact to be created, built, and uploaded.

1. Make sure you are signed in to GitHub
1. Click on the **Checks** tab on the PR
1. On the left side, click on the bottom **PR** tab
1. Click on the **A32NX** download link at the bottom of the page
